app/src/main/java/xterminators/spellingbee/model/HighScores.java

```diff
@@ -0,0 +1,148 @@
+package xterminators.spellingbee.model;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.Scanner;
+import java.util.TreeMap;
+import java.util.Map;
+import java.util.Collections;
+
+import java.lang.reflect.Type;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class HighScores {
+
+    private static final int MAX_STORED_SCORES = 10;
```

Comment:
Why only store 10? Could we store more and dynamically only show N?

Not needed for now, but worth considering for final code.